src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -74,9 +74,12 @@ public static class MachineOptions {
         @Config.Comment({"Steam to EU multiplier for Steam Multiblocks.", "1.0 means 1L Steam -> 1 EU. 0.5 means 2L Steam -> 1 EU.", "Default: 0.5"})
         public double multiblockSteamToEU = 0.5;
 
-        @Config.Comment({"Whether machines should explode when overloaded with power.", "Default: true"})
+        @Config.Comment({"Whether machines or boilers should explode when overloaded with power or met special conditions.", "Default: true"})
```

Comment:
Could you change this config option to say "... explode causing terrain damage when overloaded ..."